ui/app/AppLayouts/Communities/controls/PermissionTypes.qml

```diff
@@ -32,8 +32,12 @@ QtObject {
         return ""
     }
 
-    function getJoinEligibilityHint(type) {
-        const template = qsTr("You are eligible to join as")
+    function getJoinEligibilityHint(type, isEditMode = false, isDirty = false) {
+        var template = qsTr("You are eligible to join as")
```

Comment:
```suggestion
        const template = qsTr("You are eligible to join as")
```

Reply:
Shoudn't the text be always in it's full form? Like `"You are eligible to join as TokenMaster"`. Bc depending on the language the position of `TokenMaster` may be different :thinking:

Reply:
You are right, it should be but we have the image in the middle unfortunately :/ And yeah, I tried with a HTML `<img>` tag but it looked really bad

Reply:
Also RTL languages will have it wrong... Do we _really_ have to display the image in the middle of the sentence @benjthayer ?

Reply:
Displaying it at the beginning/end would fix this problem